example_code/pshmem_no_weak_symbol_2.c

```diff
@@ -1,2 +0,0 @@
-void SHFN(shmem_example)(/* appropriate arguments */) { /* function body */
-}
```

Comment:
Is this file empty now? Can't tell from the web view.

Reply:
Yes, the file was deleted.